client/src/app/ui/modules/editor/components/editor/editor.component.scss

```diff
@@ -15,7 +15,7 @@
             margin-right: 4px;
             margin-bottom: 4px;
             border-radius: 4px;
-            color: var(--mdc-text-button-label-text-color, inherit);
```

Comment:
Did you have a look if there is another variable you could utilize for that instead of hard coding the value?

Reply:
I didn't find any matching theme color, but `--mat-menu-item-icon-color` seem to be the same.
Should use that instead?